Externals/SlippiLib/SlippiGame.cpp

```diff
@@ -1,560 +1,567 @@
-#include <string>
 #include <codecvt>
 #include <locale>
+#include <string>
 
 #include "SlippiGame.h"
 
 namespace Slippi {
 
-  //**********************************************************************
-  //*                         Event Handlers
-  //**********************************************************************
-  //The read operators will read a value and increment the index so the next read will read in the correct location
-  uint8_t readByte(uint8_t* a, int& idx, uint32_t maxSize, uint8_t defaultValue) {
-    if (idx >= (int)maxSize) {
-      idx += 1;
-      return defaultValue;
-    }
-
-    return a[idx++];
+//**********************************************************************
+//*                         Event Handlers
+//**********************************************************************
+// The read operators will read a value and increment the index so the next read
+// will read in the correct location
+uint8_t readByte(uint8_t *a, int &idx, uint32_t maxSize, uint8_t defaultValue) {
```

Comment:
I don't know what exactly you ran but I don't think this was done correctly. Looking at clang format and description of dolphin code style, PointerAlignment is supposed to be on the left but your output places it on the right

Reply:
hm, let me take a look.
FWIW this line is on their contributing.md:
`In case of conflicts between this guide and clang-format rules, the latter should be followed instead of this guide.`

Reply:
as you mentioned this was due to .clangformat file not present in the Externals dir. Updated these files with proper formatting.

Reply:
we decided not to add the .clangformat file in this PR since these files should be moved under the Source directory